allennlp/service/predictors/wikitables_parser.py

```diff
@@ -36,3 +57,42 @@ def _json_to_instance(self, json_dict: JsonDict) -> Tuple[Instance, JsonDict]:
                                                          tokenized_question=tokenized_question)
         extra_info = {'question_tokens': tokenized_question}
         return instance, extra_info
+
+    @overrides
+    def predict_json(self, inputs: JsonDict, cuda_device: int = -1) -> JsonDict:
+        instance, return_dict = self._json_to_instance(inputs)
+        outputs = self._model.forward_on_instance(instance, cuda_device)
+
+        logical_form_filename = os.path.join(SEMPRE_DIR, 'logical_forms.txt')
+        with open(logical_form_filename, 'w') as f:
+            f.write(outputs['logical_form'] + '\n')
+
+        table_dir = SEMPRE_DIR + 'csv/'
+        os.makedirs(table_dir, exist_ok=True)
+        table_filename = 'context.csv'
+        with open(table_dir + table_filename, 'w', encoding='utf-8') as f:
+            f.write(inputs["table"])
+
+        test_record = ('(example (id %s) (utterance %s) (context (graph tables.TableKnowledgeGraph %s))'
```

Comment:
Adding a comment here saying that we need to get the utterance and the table into SEMPRE, but the target value will be ignored, would be good.

Reply:
done.